presto-docs/src/main/sphinx/sql/create-table.rst

```diff
@@ -84,6 +84,18 @@ plus additional columns at the start and end::
       another_orderdate date
     )
 
+Create the table ``orders`` and partition the data by the content of
+the ``orderdate`` column::
+    CREATE TABLE orders (
+      orderkey bigint,
+      orderstatus varchar,
+      totalprice double,
+      orderdate date
+    )
+    WITH (partitioned_by = ARRAY['orderdate']);
```

Comment:
partition by 2 columns as from this reader does not know whether it is ARRAY['orderdate','orderstatus'] or ARRAY['orderdate,orderstatus'] or ARRAY['orderdate'],ARRAY['orderstatus']

Reply:
thats a good idea.. you then need to change the status column to be lower in the DDL as well.